doc/classes/ResourceImporterWAV.xml

```diff
@@ -4,13 +4,14 @@
 		Imports a WAV audio file for playback.
 	</brief_description>
 	<description>
-		WAV is an uncompressed format, which can provide higher quality compared to Ogg Vorbis and MP3. It also has the lowest CPU cost to decode. This means high numbers of WAV sounds can be played at the same time, even on low-end deviceS.
+		WAV is an uncompressed format, which can provide higher quality compared to Ogg Vorbis and MP3. It also has the lowest CPU cost to decode. This means high numbers of WAV sounds can be played at the same time, even on low-end devices.
+		By default, Godot imports WAV files using the lossy QOA compression. You may change this by setting the [member compress/mode] property.
 	</description>
 	<tutorials>
 		<link title="Importing audio samples">$DOCS_URL/tutorials/assets_pipeline/importing_audio_samples.html</link>
 	</tutorials>
 	<members>
-		<member name="compress/mode" type="int" setter="" getter="" default="0">
+		<member name="compress/mode" type="int" setter="" getter="" default="2">
 			The compression mode to use on import.
 			[b]Disabled:[/b] Imports audio data without any compression. This results in the highest possible quality.
 			[b]RAM (Ima-ADPCM):[/b] Performs fast lossy compression on import. Low CPU cost, but quality is noticeably decreased compared to Ogg Vorbis or even MP3.
```

Comment:
That's not necessary to solve in this PR, but we should think about how to better document the int value of editor-side enums in the docs @Calinou @Mickeon. The current description is great for the tooltips in the editor, but when reading the class reference itself, it doesn't tell users how to set these values (which I assume can be 
done with `set("compress/mode", 2)` for example).

We _probably_ don't want to expose full blown enum constants for each of these config enums, but documenting it in the class ref should be good.

Could be e.g.:
```
[b]Disabled ([code]0[/code]):[/b] Imports audio data without any compression. This results in the highest possible quality.
```
(not sure `code` within `b` will work)

Or:
```
[b]Disabled:[/b] Imports audio data without any compression. This results in the highest possible quality. (value: [code]0[/code])
```
(a bit awkward though)

Ideas welcome.

Reply:
I have already thought about it in the past. There's no rule set-in-stone for these.

My own way, and what I would stick to, is what I did in https://github.com/godotengine/godot/pull/86977.

![image](https://github.com/user-attachments/assets/5e2a3dcb-4d10-4503-abca-af39d30b90d2)


Do note that someday we *will* support actual lists in the documentation ([one way](https://github.com/godotengine/godot/pull/67037) or [another](https://github.com/godotengine/godot/pull/88900)) so these won't look excessively spaced apart.